src/pages/BookingList/BookingFormat.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaAngleRight } from 'react-icons/fa';
+import img from '../../assets/images/logo.jpg';
+
+const BookingFormat = () => {
+  return (
+    <ReservCard>
+      <Img>
+        <span />
+      </Img>
+      <Right>
+        <Tags>
+          <span>예약확정</span>
+        </Tags>
+        <Content fontSize="18px">낙성대 모임 트그-스터디룸,회의실</Content>
+
+        <Ptag color="#656565">2022.05.04 (수) 12시 ~ 15시, 3시간</Ptag>
+        <Ptag margin="12px">18,000</Ptag>
+      </Right>
+      <FaAngleRight size="22px" />
+    </ReservCard>
+  );
+};
+
+export default BookingFormat;
+
+const ReservCard = styled.a`
+  display: flex;
+  position: relative;
+  width: calc(100% / 2 - 10px);
+  padding: 20px;
+  margin: 5px;
+  box-shadow: none;
+  border: 1px solid #ebebeb;
+  cursor: pointer;
+
+  svg {
+    position: absolute;
+    right: 0px;
+    bottom: calc(130px / 2);
+    color: rgba(0, 0, 0, 0.3);
+  }
+`;
+
+const Tags = styled.div`
+  display: flex;
+  margin-bottom: 10px;
+
+  span {
+    height: 23px;
+    padding: 3px 10px;
+    background-color: ${props => props.theme.purple};
+    border: none;
+    color: #fff;
+    border-radius: 15px;
+    line-height: 20px;
+    font-size: 12px;
+    font-weight: 400;
+  }
+`;
+
+const Content = styled.h3`
+  padding-bottom: 5px;
+  font-size: 18px;
+  font-weight: 800;
+`;
+
+const Ptag = styled.p`
+  margin-top: ${props => props.margin};
+  color: ${props => props.color};
+  font-size: ${props => props.fontSize};
```

Comment:
스타일드 컴포넌트 잘 활용해 주셨네요!
이부분도 구조분해 할당 가능합니다:)
```suggestion
const Ptag = styled.p`
  margin-top: ${({ margin }) => margin};
  color: ${({ color }) => color};
  font-size: ${({ fontSize }) => fontSize};
`;
```